website/docs/zh/config/dev/watch-files.mdx

```diff
@@ -16,6 +16,28 @@ type WatchFiles = {
 
 如果 `dev.hmr` 和 `dev.liveReload` 都设置为 false，则 `watchFiles` 将被忽略。
 
+### 前置知识
+
+Rsbuild (Rspack) 默认会监听所有构建依赖，当这些文件发生变化时，会触发重新编译。当你希望有一些**非构建依赖的文件**发生变化时，页面可以重新加载，可以配置此配置项。
+
+需要注意的是，WatchFiles 中文件发生变化时，不会触发配置文件的重新加载及重新构建编译。
+
+同时，如果你希望某个**构建依赖的文件**变化时，不触发构建的重新编译，可以使用 Rspack 的 [watchOptions.ignored](https://www.rspack.dev/zh/config/watch#watchoptionsignored) 配置项。
+
+例如，当你希望你的项目在 node_modules 中文件发生变化时，不触发重新构建编译，可以如下配置：
+
+```js
+export default {
+  tools: {
+    bundlerChain: (chain) => {
```

Comment:
I prefer using `tools.rspack` instead of `tools.bundlerChain` in our examples, it is easier to use.